src/dotless.Test/Specs/RulesetsFixture.cs

```diff
@@ -135,5 +135,135 @@ public void IESlash9HackAllowedForNonEvaluatedValues()
             var expected = input;
             AssertLess(input, expected);
         }
+
```

Comment:
Having the tests in RulesetsFixture is fine by me -- what I'd like to see is a test or two covering cases where !important is involved.